src/test/java/com/codingnomads/impacttracker/logic/CommitmentServiceTest.java

```diff
@@ -56,4 +59,30 @@ public void whenGettingCommitmentFromUserId_shouldReturnListOfCommitments() {
         assertThat(commitmentList).contains(commitment1,commitment2);
     }
 
+    @Test
+    public void whenUpdatingCommitmentById_shouldReturnUpdatedCommitment(){
+        when(mySqlCommitmentRepositoryMock.updateCommitmentById(1, commitment3)).thenReturn(commitment1);
```

Comment:
I would rename `commitment1` to `returnedCommitment` and `commitment3` as `inputCommitment`, Its easier to read like that

Reply:
thanks- I'll do that!